qa/Gemfile

```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
+gem "runner-tool", :git => "https://github.com/purbon/runner-tool.git"
+gem "rspec", "~> 3.1.0"
+gem "rake"
+gem "pry", :group => :test
+gem "stud"
+gem "logstash-devutils"
```

Comment:
actually we can remove logstash-devutils for now, we just removed in https://github.com/elastic/logstash/pull/5350/commits/f532edbe108340457e391e2d05da8ddfa6568018 the only dependency we had.

For the other's I think we're safe in this side, we could also remove stud as is not used anywhere.